docs/usage/after_provisioning.md

```diff
@@ -298,6 +298,12 @@ The current settings can be seen with in the `diag` helper output or with `chass
 $ ipmi aa:bb:cc:dd:ee:ff raw 0x0 0x8 0x3 0x1f
 ```
 
+All subcommands that are not interactive like, for example, the serial console, can also run in batch mode for all servers:
```

Comment:
```suggestion
All sub commands that are not interactive (example: the serial console) can also run in batch mode for all servers:
```

Reply:
Taken in a different form: _All sub commands that are non-interactive, i.e., almost everything except the serial console,_